stderr of test 'tst161` in directory 'monetdb5/mal` itself:


# 19:09:44 >  
# 19:09:44 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=43514 --set sql_port=57655 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst161.mal </dev/null
# 19:09:44 >  

MAPI  = (monetdb) /var/tmp/mtest-13954/.s.monetdb.38078
QUERY = # testing definition of new atom type based on ptr
        
        atom mytype:ptr;
        
        function l():mytype;
            return l:= 0:mytype;
        end l;
        
        z:= mytype.l();
        io.print(z);
ERROR = !SyntaxException:defConstant:constant coercion error from int to mytype
        !TypeException:user.main[2]:'mytype.l' undefined in:     z:mytype := mytype.l();

# 13:20:34 >  
# 13:20:34 >  "Done."
# 13:20:34 >  

